### DISCUSSION TITLE
creating a new aircraft

### DISCUSSION THREAD

**Discussion by gcasweb** [JSBSim-Team/jsbsim#1068] [General]
Hi everyone!

I'm making a new aircraft project and I want to transfer it to JSBsim and simulate it on Flight Gear. And my doubt is how to, using JSBsim, make a model. what is (or are) the first(s) step(s) to do it?

Tnx for reading

**Comment by bomber64** [comment]
Mass Ballance to establish inertia, CoG, empty and all up weight.
